docs/models-usage.md

```diff
@@ -2,32 +2,36 @@
 
 ## Data retrieval / Finders
 
-Finder methods are intended to query data from the database. They do *not* return plain objects but instead return model instances. Because finder methods return model instances you can call any model instance member on the result as described in the documentation for [*instances*](instances.html).
+Finder methods are intended to query data from the database. They do _not_ return plain objects but instead return model instances. Because finder methods return model instances you can call any model instance member on the result as described in the documentation for [_instances_](instances.html).
 
 In this document we'll explore what finder methods can do:
 
 ### `find` - Search for one specific element in the database
 
 ```js
-// search for known ids
+// search for known id
 Project.findByPk(123).then(project => {
   // project will be an instance of Project and stores the content of the table entry
   // with id 123. if such an entry is not defined you will get null
-})
+});
+
+// search for known array of ids
+Project.findByPks([1, 2, 3]).then(project => {
```

Comment:
This will only work if the user installs your plugin, so this should not be present in the sequelize main documentation, please remove this, this PR will become a docs-only change